app/controllers/state_file/questions/confirmation_controller.rb

```diff
@@ -9,7 +9,7 @@ def show_xml
                            when "az"
                              SubmissionBuilder::Ty2022::States::Az::IndividualReturn.build(submission)
                            end
-        builder_response.errors.present? ? render(plain: builder_response.errors.join("\n")) : render(xml: builder_response.document)
+        builder_response.errors.present? ? render(plain: builder_response.errors.join("\n") + "\n\n" + builder_response.document.to_xml) : render(xml: builder_response.document)
```

Comment:
Just noticing that you kept this in. I think it will be helpful for others too.

Reply:
Agreed